src/content/whats-new/2022/03/whats-new-02-28-EKS-Add-on.md

```diff
@@ -0,0 +1,19 @@
+---
+title: 'Automate Kubernetes observability on your EKS clusters with code'
+summary: 'Deploy observability faster and easier than ever!'
+releaseDate: '2022-02-28'
+learnMoreLink: 'https://newrelic.com/blog/nerdlog/eks-ssp-kubernetes-observability' 
+getStartedLink: 'https://www.npmjs.com/package/@newrelic/newrelic-ssp-addon'
+
+---
+
```

Comment:
Remove space

Reply:
@daynaslord, I removed this blank line in the front matter.